src/main.tsx

```diff
@@ -1,5 +1,20 @@
+import { BasePanel } from "./components/panel/side-panel";
+import {
+    BasePanelBody,
+    BasePanelContent,
+} from "./components/panel/side-panel.styles";
```

Comment:
components can be barrel exported in `panel/` and `components/` folders

actually there should be another component to manage which panel to display (elements, pages, details) and that should hold `BasePanelBody` and `BasePanelContent`.however to avoid scope creep, you can move these components into that component and skip the logic to decide which panel to display.